scripts/asm-installer/tests/run_basic_instance_template_creation

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -CeEu
+set -o pipefail
+
+SPATH="$(readlink -f "$0")"
+SDIR="$(dirname "${SPATH}")"; export SDIR;
+SCRIPT_NAME="${0##*/}"
+
+# shellcheck source=common.sh
+. "${SDIR}/common.sh"
+
+cd "${SDIR}"
+
+main() {
+  # CLI setup
+  parse_args "$@"
+
+  run_basic_test "install" "mesh_ca" "-o vm -o hub-meshca --enable-registration --enable-gcp-components"
```

Comment:
Why are these --enable flags necessary?

Reply:
Yea, i see it is not needed. Removed from VM control plane test as well.

Reply:
Thanks!